packages/admin-ui/components/pages/setup/dataSources/BigQueryProperties.tsx

```diff
@@ -82,6 +82,32 @@ export default function BigQueryProperties() {
       >
         <Input placeholder="Project-123" />
       </Form.Item>
+      <Form.Item
+        label="Dataset"
+        required
+        name="dataset"
+        rules={[
+          {
+            required: true,
+            message: ERROR_TEXTS.CONNECTION.DATASET.REQUIRED,
+          },
+        ]}
+      >
+        <Input placeholder="natality" />
+      </Form.Item>
+      <Form.Item
+        label="Location"
+        required
+        name="location"
+        rules={[
+          {
+            required: true,
+            message: ERROR_TEXTS.CONNECTION.LOCATION.REQUIRED,
+          },
+        ]}
+      >
+        <Input placeholder="US" />
+      </Form.Item>
```

Comment:
Is it can put a guide link like https://cloud.google.com/bigquery/docs/locations

Reply:
I just thought about...maybe not, since we added the setup guide link, next to the 'Learn BigQuery', the setup guide should have these parts.

Reply:
The different presented way is OK to me, we can check with @wwwy3y3 later.